conans/test/command/export_linter_test.py

```diff
@@ -154,3 +154,44 @@ def deploy(self):
                          client.out)
         self.assertNotIn("WARN: Linter. Line 8: self.copy_deps is not callable",
                          client.out)
+
+    def python_reuse_test(self):
```

Comment:
Not necessary a new test, there are some existing tests that will fire this warning.

Reply:
I think the purpose of the test was more clear in the *export_linter_test.py*. I would change it